src/status_im2/contexts/chat/messages/composer/view.cljs

```diff
@@ -215,15 +218,19 @@
               [edit/edit-message-auto-focus-wrapper text-input-ref edit #(clean-and-minimize params)]
               [reply/reply-message-auto-focus-wrapper text-input-ref reply]
               [rn/view
-               {:style {:height (- max-y (- min-y 38))}}
+               {:style (merge
+                        {:height (- max-y (- min-y 38))}
+                        (when (and @input/recording-audio? reply)
+                          {:z-index -1}))}
```

Comment:
When recording an audio and replying at the same time, text input is overlapped by the reply component but still have priority over touches, so we need to make sure reply component receives the touches in this scenario